src/components/Button.jsx

```diff
@@ -1,16 +1,50 @@
-const Button = ({ children, icon, backgroundColor, textColor, borderColor, fullWidth }) => {
-	return (
-		<button
-			className={`flex justify-center items-center gap-2 px-7 py-4 border font-montserrat text-lg leading-none rounded-full ${
-				backgroundColor ? `${backgroundColor} ${borderColor} ${textColor}` : "bg-coral-red  border-coral-red text-white"
-			}
-			${fullWidth && 'w-full'}
+const Button = ({
+  children,
```

Comment:
- We do not recommend using direct CSS styling for this project.
- Read the Tailwind CSS docs and then add hover effects to this button component

Reply:
I have added the hover effects according to Tailwind CSS.

Reply:
Great work!